core/tests/protractor_utils/AdminPage.js

```diff
@@ -292,36 +327,111 @@ var AdminPage = function() {
     }
   };
 
-  this.addReviewer = async function(name, newRole) {
-    await action.click('Admin Roles Tab', adminRolesTab);
-    // Change values for "add reviewer for community" form, and submit it.
-    await action.sendKeys('Add reviewer name', addReviewerName, name);
-    await action.sendKeys('Select reviewer role', selectReviewerRole, newRole);
-    await action.click('Add Reviewer button', addReviewerFormSubmitButton);
+  var _assignReviewer = async function(
+      username, reviewCategory, languageDescription = null) {
+    await _switchToRolesTab();
+
     await waitFor.visibilityOf(
-      statusMessage, 'Confirmation message not visible');
+      assignReviewerForm, 'Assign reviewer form is not visible');
+
+    var usernameInputField = assignReviewerForm.element(reviewerUsernameCss);
+    await action.sendKeys(
+      'Username input field', usernameInputField, username);
+
+    var reviewCategorySelectField = assignReviewerForm.element(
+      reviewCategorySelectCss);
+
+    await action.select(
+      'Review category selector', reviewCategorySelectField, reviewCategory);
+
+    if (languageDescription !== null) {
+      var languageSelectField = assignReviewerForm.element(languageSelectCss);
+
+      await action.select(
+        'Language selector', languageSelectField, languageDescription);
+    }
+
+    var submitButton = assignReviewerForm.element(reviewerFormSubmitButtonCss);
+
+    await action.click('Submit assign reviewer button', submitButton);
```

Comment:
Nit: There's a lot of new line spacing here—consider consolidating some lines? Here and below.

Reply:
Done!